modules/fetching-vulnerability-definitions.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images.adoc
+:_module-type: CONCEPT
+[id="fetching-vulnerability-definitions_{context}"]
+= Fetching vulnerability definitions
+
+Scanner fetches the vulnerability definitions every 5 minutes from a single feed.
+This feed combines vulnerability definitions from upstream sources that include multiple Linux distributions and the National Vulnerability Database, and it refreshes every hour.
+
+* The address of the feed is `\https://definitions.stackrox.io`.
```

Comment:
link:\https://definitions.stackrox.io[Stackrox defintions] or something similar. We use links for external websites.

Reply:
@stevsmit This link is inline code and I don't want it to appear as a link and that's why I escaped it. 

We can't use `link:` here as it renders as `link:\https://definitions.stackrox.io`.  No changes made.